Sources/Sentry/include/SentryUIDeviceWrapper.h

```diff
@@ -0,0 +1,20 @@
+#import "SentryDefines.h"
+
+#if SENTRY_HAS_UIKIT
+#    import <UIKit/UIKit.h>
+#endif
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SentryUIDeviceWrapper : NSObject
+
+#if TARGET_OS_IOS
```

Comment:
`m`: If all that only works on iOS, I think we could wrap the whole class in this compiler directive.